daemon/prune.go

```diff
@@ -140,7 +140,7 @@ func (daemon *Daemon) VolumesPrune(ctx context.Context, pruneFilters filters.Arg
 			if err != nil {
 				logrus.Warnf("could not determine size of volume %s: %v", name, err)
 			}
-			err = daemon.volumes.Remove(v)
+			err = daemon.VolumeRm(v.Name(), false)
```

Comment:
Looks like there's a `VolumeRm` and a `volumeRm` where the former takes care of the bool part and the latter takes care of the rest.
Maybe we can change `volumeRm` to take a `volume.Volume` instead of a name so we don't have to re-lookup the volume and then for `VolumesRm` it can just do it's own lookup.

Reply:
@ripcurld0 looks like you updated the function but forgot to update this; should this now be like below?

```go
err = daemon.volumeRM(v)
```

Reply:
@thaJeztah I think you meant `@sterchelen`.
As @cpuguy83 mentioned:

> Maybe we can change **volumeRm** to take a _volume.Volume_ instead of a name so we don't have to re-lookup the volume and then for **VolumesRm** it can just do **it's own lookup**.

So, if I well understood, we just have to change the `volumeRm` function to take a `volume.Volume` instead of the name and let the actual `VolumesRm` function signature.

Reply:
@thaJeztah I don't remember changing that line. However, `git blame` says it was @mlaventure 😆